### PR TITLE
add optional use_tls parameter

### PR DESCRIPTION
**Pull request by MartinHelwig**
##### SUMMARY

This adds the possibility to use encrypted communication.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

znode
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/znode.py [hunk 1]
```diff
@@ -208,7 +216,7 @@ def check_params(params):
 class KazooCommandProxy():
     def __init__(self, module):
         self.module = module
-        self.zk = KazooClient(module.params['hosts'])
+        self.zk = KazooClient(module.params['hosts'], use_ssl=module.params['use_tls'])
```
Comment (felixfontein) [inline plugins/modules/znode.py:219]: Do all versions of kazoo >= 2.1 support this?

Comment (MartinHelwig) [reply plugins/modules/znode.py:219]: As far as I know, yes.

**Comment by felixfontein** [comment]
Looks good to me. If nobody objects, I'll merge this in a week.

**Comment by felixfontein** [comment]
@MartinHelwig thanks for your contribution!

**Pull request merged**